core/src/test/java/org/easymock/tests2/EasyMockAutoInstantiationTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2001-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.easymock.tests2;
+
+import org.easymock.*;
+import org.easymock.tests.IMethods;
+import org.easymock.tests.IVarArgs;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Tests automated instantiations of {@link org.easymock.TestSubject}
+ *
+ * @author Tatsuya Iwanari
+ */
+@RunWith(EasyMockRunner.class)
+public class EasyMockAutoInstantiationTest extends EasyMockSupport {
```

Comment:
Some lines were copied from https://github.com/easymock/easymock/blob/master/core/src/test/java/org/easymock/tests2/EasyMockAnnotationsTest.java to make sure that this doesn't change the previous behavior but it would look noisy because of the many arguments.